lib/web/join_tokens.go

```diff
@@ -211,11 +210,22 @@ func (h *Handler) createTokenHandle(w http.ResponseWriter, r *http.Request, para
 func (h *Handler) getNodeJoinScriptHandle(w http.ResponseWriter, r *http.Request, params httprouter.Params) (interface{}, error) {
 	httplib.SetScriptHeaders(w.Header())
 
+	var autoUpgradesVersion string
+	var err error
+	autoUpgrades := automaticUpgrades(h.ClusterFeatures)
+	if autoUpgrades {
+		autoUpgradesVersion, err = h.cfg.AutomaticUpgradesChannels.DefaultVersion(r.Context())
+		if err != nil {
+			return nil, trace.Wrap(err, "failed to get auto-upgrade version")
```

Comment:
Should we also send an error message here if we fail to generate the install script?
```
log.WithError(err).Info("Failed to return the node install script.")
w.Write(scripts.ErrorBashScript)
```